testdata/p4_16_samples_outputs/pna-add-on-miss.p4.spec

```diff
@@ -60,9 +57,8 @@ action next_hop2 args instanceof next_hop2_arg_t {
 }
 
 action add_on_miss_action2 args none {
-	mov m.MainControlT_tmp 0x0
 	mov m.MainControlT_tmp_0 0x4d2
-	learn next_hop2 m.MainControlT_tmp m.local_metadata_timeout
+	learn next_hop2 0x0 m.local_metadata_timeout
```

Comment:
This is incorrect. Learn instruction doesn't accept immediate

Reply:
ok

Reply:
done